client/src/components/website/ExplanationText.tsx

```diff
@@ -532,47 +541,37 @@ if ( roomIdFromUrl != null ) {
   const MAX_CONNECT_ATTEMPTS = 50;
   const TRY_CONNECT_INTERVAL_MS = 1000;
 
-  const lobbyInfo = await lobbyClient.getLobbyInfo(appId, roomId);
-
-  if (lobbyInfo.visibility === "local") {
```

Comment:
why do we want to remove this?

Reply:
This section of the docs is already massive (has 3 code blocks), felt this was excessive and not relevant to the core functionality trying to be shown.

Also, LobbyV3 actually broke that code path. Harsh actually just merged a fix for Local Lobbies where we have a Local Room now, so will have to update BulletMania eventually to incorporate that. https://github.com/hathora/hathora-cloud-monorepo/pull/1542

Reply:
gotcha sounds good